config/settings.yml

```diff
@@ -149,6 +149,9 @@ locators:
   vha_access_waittime: https://www.accesstopwt.va.gov/
   base_path: https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/
 
+ppms:
+  url: https://some.fakesite.com
```

Comment:
I'm guessing there isn't a dev url that can be used here?

Reply:
There is a dev url, but it can only be accessed from within the VA network. Furthermore their documentation states "In the future, the Data Web Service will be secured and require all consumers to request access to the endpoint" so I'm unsure if it should be included here. Currently, there's a special forward proxy url that gets set in settings.local.yml in the dev/review instance environments that connects to the the dev ppms dws with similar behavior happening for staging and the nonprod ppms.